libraries/math/src/instruction.rs

```diff
@@ -167,3 +167,72 @@ pub fn noop() -> Instruction {
         data: MathInstruction::Noop.try_to_vec().unwrap(),
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_precise_sqrt() {
+        let instruction = precise_sqrt(u64::MAX);
+        assert_eq!(0, instruction.accounts.len());
+        assert!(!instruction.data.is_empty());
```

Comment:
For most of the pack / unpack tests, we test against what's actually expected in the buffer, so something like:
```
assert_eq!(instruction.data, [0, 255, 255, 255, 255, 255, 255, 255, 255]);
```
Alternatively, you can do:
```
assert_eq!(instruction.data, MathInstruction::PreciseSquareRoot { radicand: u64::MAX }.try_to_vec().unwrap());
```